src/caselawclient/models/identifiers/press_summary_ncn.py

```diff
@@ -0,0 +1,20 @@
+from .neutral_citation import NeutralCitationNumber, NeutralCitationNumberSchema
+
+
+class PressSummaryRelatedNCNIdentifierSchema(NeutralCitationNumberSchema):
+    """
+    Identifier schema for relating a Press Summary to a Judgment with a given NCN
+    """
+
+    name = "Press Summary relates to NCN"
+    namespace = "psrelatedncn"
```

Comment:
Are we happy this is the right namespace name? others have been uk / fcl prefixed which felt like a category